### DISCUSSION TITLE
Errors after update Visual Studio to 17.2.0 Preview 4.0

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#8571] [Q&A]
![изображение](https://user-images.githubusercontent.com/69314237/164087485-47628652-6732-48fc-9d1d-0cbc329d8d66.png)

Full error details:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not copy the file "obj\x64\Debug\C:\Users\limeniye\Desktop\ligric\LigricView\View\LigricUno.Shared\Views\Pages\Login\LoginPage.xbf" because it was not found.	LigricUno.UWP			
```

![изображение](https://user-images.githubusercontent.com/69314237/164087908-7334f4f8-22a1-4fbe-91ab-c86e9cf4a6a5.png)

Updated:
I have .gitignore in my github repo. Therefore, I deleted my project and cloned it again. But it didn't help me.

**Comment by jeromelaban** [comment]
Thanks, see this discussion for a workaround: https://github.com/unoplatform/uno/discussions/5007
